tests/tasks/verify_settings.yml

```diff
@@ -1,49 +1,14 @@
 - name: Verify password
   when: __verify_mssql_password is defined
   block:
-    - name: Ensure that the mssql-server service is started
-      service:
-        name: mssql-server
-        state: started
-
-    - name: Check if a custom tcpport setting exist
-      command: /opt/mssql/bin/mssql-conf get network tcpport
-      ignore_errors: true
-      changed_when: false
-      register: __mssql_custom_tcp_port
-
-    - name: Check if a custom ipaddress setting exist
-      command: /opt/mssql/bin/mssql-conf get network ipaddress
-      ignore_errors: true
-      changed_when: false
-      register: __mssql_custom_ip_address
-
-    - name: Check if the set password matches the existing password
+    - name: Log in using the set password
+      include_tasks: ../../tasks/verify_password.yml
```

Comment:
because of the differences between the installed roles environment and the installed collections environment, you'll have to use `include_role:` with `tasks_from:` here in the test: https://docs.ansible.com/ansible/2.8/modules/include_role_module.html#include-role-module
```yaml
     - name: Log in using the set password
       include_role:
         name: linux-system-roles.mssql
         tasks_from: verify_password.yml
```

Reply:
in the role tasks/main.yml you can just use `include_tasks: verify_password.yml` directly